backend-rust/src/migrations.rs

```diff
@@ -192,14 +193,16 @@ pub enum SchemaVersion {
     AddAccumulatedPoolState,
     #[display("0008:AccountBaseAddress")]
     AccountBaseAddress,
+    #[display("0007:Add delegated stake cap")]
```

Comment:
?
```suggestion
    #[display("0009:Add delegated stake cap")]
```

Reply:
Rebased on top of main and updated the migration to version `10`.